src/dmd/frontend.h

```diff
@@ -1703,7 +1703,7 @@ struct ParameterList
     Array<Parameter*>* parameters;
     StorageClass stc;
     VarArg varargs;
-    ParameterList(Array<Parameter*>* parameters, VarArg varargs = cast(VarArg)cast(ubyte)0u, StorageClass stc = 0);
+    ParameterList(Array<Parameter*>* parameters, VarArg varargs = VarArg.none, StorageClass stc = 0);
```

Comment:
This isn't valid C++, it's `VarArg::none`.

Reply:
CC @MoonlightSentinel

Reply:
Access to enum/static members needs more work but that's unrelated to this PR. (`frontend.h` doesn't compile anyways)